scripts/run-integration-template.bash

```
@@ -1,11 +1,11 @@
 #!/bin/bash
 
-pushd () {
-    command pushd "$@" > /dev/null
+pushd() {
+    command pushd "$@" >/dev/null
```

Shellcheck (0.10.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)